openstack/ironic/templates/seed.yaml

```diff
@@ -101,7 +101,7 @@ spec:
       "capabilities:cpu_arch": "x86_64"
       "quota:separate": "true"
       "quota:instance_only": "true"
-#      "resources:CUSTOM_ZG1BCM1_MEDIUM": "1"
+      "resources:CUSTOM_ZG1BCM1_MEDIUM": "1"
```

Comment:
This is the same as above. It shouldn't be, shouldn't it? There seems to be a `1` missing.